statistics/feedback.go

```diff
@@ -727,3 +745,16 @@ func (q *QueryFeedback) recalculateExpectCount(h *Handle) error {
 	q.expected = int64(expected)
 	return nil
 }
+
+// splitFeedback splits the feedbacks into equality feedbacks and range feedbacks.
+func splitFeedbackByQueryType(feedbacks []feedback) ([]feedback, []feedback) {
+	var eqFB, ranFB []feedback
+	for _, fb := range feedbacks {
+		if bytes.Compare(kv.Key(fb.lower.GetBytes()).PrefixNext(), fb.upper.GetBytes()) >= 0 {
```

Comment:
how about `== 0`?

Reply:
or `bytes.Equal`

Reply:
No, we cannot. Because sometimes the `lower` is equal to `upper`, so use `==0` would miss some feedbacks.